tests/protractor/priority/priority.js

```diff
@@ -26,24 +23,21 @@ app.controller('PriorityCtrl', function Chat($scope, $firebase) {
     if ($scope.message !== '') {
       // Add a new message to the messages list
       var priority = $scope.messages.length;
-      $scope.messages.$inst().$push({
+      $scope.messages.$add({
         from: $scope.username,
         content: $scope.message
       }).then(function (ref) {
-        var newItem = $firebase(ref).$asObject();
+        var newItem = new $FirebaseObject(ref);
 
         newItem.$loaded().then(function (data) {
           verify(newItem === data, '$FirebaseArray.$loaded() does not return correct value.');
```

Comment:
Not your bug, but this comment should reference `$FirebaseObject` not `$FirebaseArray`.